internal/cmp/resource_network.go

```diff
@@ -138,12 +142,24 @@ func (r *resNetwork) Delete(ctx context.Context, d *utils.Data, meta interface{}
 	return err
 }
 
-func alignNetworkReq(request *models.CreateNetwork) {
-	request.Site.ID = request.GroupID
-	if request.NetworkDomainID != 0 {
-		request.NetworkDomain = &models.IDModel{ID: request.NetworkDomainID}
+func (r *resNetwork) networkRequest(createReq *models.CreateNetwork) error {
+	if createReq.TfDhcpNetwork != nil {
+		createReq.DhcpServer = createReq.TfDhcpNetwork.DhcpServer
+		if createReq.TfDhcpNetwork.Config != nil {
+			createReq.Config.ConnectedGateway = createReq.TfDhcpNetwork.Config.ConnectedGateway
```

Comment:
please check if each of these fields (by checking UI if these values are shown righly) are set while you are testing it

Reply:
yes its working correctly